client/session.go

```diff
@@ -1895,6 +1902,22 @@ func (b blockMetadatasByTime) Less(i, j int) bool {
 	return b[i].start.Before(b[j].start)
 }
 
+func newTimesByUnixNanos(values []int64) []time.Time {
+	result := make([]time.Time, len(values))
```

Comment:
This is only for a log output, this is an error log too so needs to be avoided in production anyhow.